package.json

```diff
@@ -15,7 +15,7 @@
     "node": ">=12"
   },
   "scripts": {
-    "test": "npm run lint && jest",
+    "test": "npm run lint && jest --collectCoverage",
```

Comment:
Was `collectCoverage: true` not working properly?

Reply:
It's a little annoying to have to specify not to collect coverage during the development loop for a test, though this is more of a personal preference and maybe I should be using jest --watch. I can revert if you like!

Reply:
I reverted